src/store/plugins/persistState.js

```diff
@@ -1,7 +1,9 @@
+import { cloneDeep } from 'lodash-es';
+
 const KEY = 'state';
 
 const setState = async state => {
-  browser.storage.local.set({ [KEY]: state });
+  browser.storage.local.set({ [KEY]: cloneDeep(state) });
```

Comment:
what was the problem there?

Reply:
There was problem with saving the state in Firefox and `cloneDeep` fixed the problem. Not sure what was the exact error.

Reply:
Putting to local storage is already a kind of cloning, this part is super unobvious.